### PR TITLE
rename all multicrit to multiobj

### PR DESCRIPTION
**Pull request by jakob-r**
This deas with issue #281.

Tests should pass. Anyhow i will check the diff again and it would be awesome if another person can check it.

### CODE REVIEW

**Comment by jakobbossek** [comment]
Going to check it now.

File: R/doc_mbo_parallel.R [hunk 1]
```diff
@@ -16,8 +16,8 @@
 #' }
 #' Details regarding the latter:
 #' \describe{
-#' \item{Singlecrit MBO with LCB multipoint}{Parallel optimization of LCBs for the lambda-values.}
-#' \item{Multicrit MBO with ParEGO}{Parallel optimization of scalarization functions.}
+#' \item{Single objective MBO with LCB multipoint}{Parallel optimization of LCBs for the lambda-values.}
+#' \item{Multi-objective MBO with ParEGO}{Parallel optimization of scalarization functions.}
```
Comment (jakobbossek) [inline R/doc_mbo_parallel.R]: Single objective, but multi-objective. There is a hyphen missing.

File: R/makeMBOControl.R [hunk 2]
```diff
@@ -5,7 +5,7 @@
 #'
 #' @param n.objectives [\code{integer(1)}]\cr
 #'   How many objectives are to be optimized? \code{n.objectives = 1} implies normal single
-#'   criteria optimization, \code{n.objectives > 1} implies multicriteria optimization.
+#'   criteria optimization, \code{n.objectives > 1} implies multiobjective optimization.
 #'   Default is 1.
```
Comment (jakobbossek) [inline R/makeMBOControl.R]: ... single criteria optimization ...

File: R/setMBOControlMultiPoint.R [hunk 3]
```diff
@@ -25,25 +25,25 @@
...
 #'   Possible values are:
 #'   \dQuote{hypervolume}: Non-dominated sorting + hypervolume contribution.
 #'   \dQuote{crowdingdist}: Non-dominated sorting + crowding distance based ranking.
 #'   \dQuote{first}: Non-dominated sorting + first objective of \code{moimbo.objective} as criterion.
 #'   \dQuote{last}: Non-dominated sorting + last objective of \code{moimbo.objective} as criterion.
 #'   Default is \code{hypervolume}.
 #' @param moimbo.maxit [\code{character(1)}]\cr
-#'   Number of generations for multicriteria EA.
+#'   Number of generations for multiobjective EA.
 #'   Default is 100.
```
Comment (jakobbossek) [inline R/setMBOControlMultiPoint.R]: multi-objective

File: R/setMBOControlTermination.R [hunk 4]
```diff
@@ -84,7 +84,7 @@ setMBOControlTermination = function(control,
 
   if (!is.null(target.fun.value)) {
     if (control$n.objectives > 1L)
-      stop("Specifying target.fun.value is only useful in single crit optimization.")
+      stop("Specifying target.fun.value is only useful in single objective optimization.")
```
Comment (jakobbossek) [inline R/setMBOControlTermination.R]: single-objective

File: R/setMBOControlTermination.R [hunk 5]
```diff
@@ -16,7 +16,7 @@
 #'   the condition is checked after each iteration.
 #'   The default \code{NULL} means: There is no execution time budget.
 #' @param target.fun.value [\code{numeric(1)}] | NULL]\cr
-#'   Termination criterion for single crit optimization: Stop if a function evaluation
+#'   Termination criterion for single objective optimization: Stop if a function evaluation
```
Comment (jakobbossek) [inline R/setMBOControlTermination.R]: single-objective

File: man/exampleRunMultiObj.Rd [hunk 6]
```diff
@@ -1,10 +1,10 @@
 % Generated by roxygen2: do not edit by hand
-% Please edit documentation in R/exampleRunMultiCrit.R
-\name{exampleRunMultiCrit}
-\alias{exampleRunMultiCrit}
+% Please edit documentation in R/exampleRunMultiObj.R
+\name{exampleRunMultiObj}
+\alias{exampleRunMultiObj}
 \title{Perform an MBO run on a multi-criteria test function and and visualize what happens.}
```
Comment (jakobbossek) [inline man/exampleRunMultiObj.Rd]: multi-objective

File: man/makeMBOControl.Rd [hunk 7]
```diff
@@ -17,7 +17,7 @@ makeMBOControl(n.objectives = 1L, propose.points = 1L,
 \arguments{
 \item{n.objectives}{[\code{integer(1)}]\cr
 How many objectives are to be optimized? \code{n.objectives = 1} implies normal single
-criteria optimization, \code{n.objectives > 1} implies multicriteria optimization.
+criteria optimization, \code{n.objectives > 1} implies multiobjective optimization.
```
Comment (jakobbossek) [inline man/makeMBOControl.Rd]: multi-objective

File: man/setMBOControlMultiPoint.Rd [hunk 8]
```diff
@@ -22,10 +21,10 @@ Possible values are:
 exp(1)-distribution, so do not define \code{infill.opt.cb.lambda}.
 The optimizer for each proposal is configured in the same way as for the single point case,
 i. e., by specifying \code{infill.opt} and related stuff.
-\dQuote{multicrit}: Proposes points by evolutionary multicriteria optimization.
-The EA is a (mu+1) type of algorithm and runs for \code{multicrit.maxit} generations.
+\dQuote{moimbo}: Proposes points by multiobjective infill criteria via evolutionary multiobjective optimization.
```
Comment (jakobbossek) [inline man/setMBOControlMultiPoint.Rd]: multi-objective

**Comment by jakobbossek** [comment]
Looks good so far. Just stick to using hyphens, i.e., always use {single,multi}-objective and do not mix up with {single,multi}objective

**Comment by berndbischl** [comment]
lol

**Comment by jakobbossek** [comment]
Consistency is important! :rage2: 😉

**Comment by jakob-r** [comment]
I changed everything to hyphens like @jakobbossek propopsed.

**Pull request merged**